test/components/Layout_test.tsx

```diff
@@ -59,6 +61,7 @@ describe("App component", () => {
       store.dispatch(sessionActions.serverInfoSuccess(serverInfo));
       store.dispatch(sessionActions.setupComplete());
       store.dispatch(sessionActions.setAuthenticated(credentials));
+      await waitFor(() => new Promise(resolve => setTimeout(resolve, 500))); // debounce wait
```

Comment:
Does `setAuthenticated()` take a few hundreds of msec so that this is necessary?